src/test/java/org/mule/extension/db/integration/storedprocedure/StoredProcedureTestCase.java

```diff
@@ -79,6 +82,16 @@ public void multipliesIntegers() throws Exception {
     assertThat(payload.get("result2").toString(), IsEqual.equalTo("60"));
   }
 
+  @Test
+  public void returnValueNull() throws Exception {
+    Map<String, Object> payload = runProcedure("returnValueNull");
+
+    // Apparently Derby has a bug: when there are no resultset returned, then
+    // there is a fake updateCount=0 that is returned. Check how this works in other DB vendors.
+    // assertThat(payload.size(), equalTo(2));
```

Comment:
Is this commentary needed for the tests, and why would you expect the updatedCount to be 2? I see that for every implementation, the stored procedure simply returns null in the OUT parameter but no record is updated for the unit test (actually no updates are needed for the unit test)